spanner_orm/field.py

```diff
@@ -43,23 +43,43 @@ def grpc_type() -> spanner_v1.Type:
   def validate_type(value: Any) -> None:
     raise NotImplementedError
 
+  @staticmethod
+  def matches(type: str) -> bool:
+    raise NotImplementedError
+
+  @staticmethod
+  def support_length() -> bool:
```

Comment:
(nit) `supports_length`, since this is returning whether or not length is supported, not enabling support for length.